flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -235,9 +235,11 @@
 
             <UseTemplate Name="FBW_AutoThrottle_Instinctive_Disconnect_Template">
                 <NODE_ID>PUSH_THROTTLE_1</NODE_ID>
+                <ANIM_NAME>PUSH_THROTTLE_1</ANIM_NAME>
             </UseTemplate>
             <UseTemplate Name="FBW_AutoThrottle_Instinctive_Disconnect_Template">
                 <NODE_ID>PUSH_THROTTLE_2</NODE_ID>
+                <ANIM_NAME>PUSH_THROTTLE_2</ANIM_NAME>
```

Comment:
You have flagged the comment related to this as resolved and indicated you'd remove these. I'll ask the question again:

Why were these elements added?

Reply:
I remove ID
I changed template so ANIM need onthere